### PR TITLE
9490 sorting adc section messages

### PR DESCRIPTION
**Pull request by jamesobrooks**

### CODE REVIEW

File: web-client/integration-tests/adcClerkViewsSectionMessagesJourney.test.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    const expected = [message5Subject, message4Subject];
+
+    expect(afterOutboxMessageCount).toEqual(
+      expected.length + beforeOutboxMessageCount,
+    );
+
+    validateMessageOrdering(outboxMessages, expected);
+  });
+
+  it('go to section completed', async () => {
```
Comment (rachelschneiderman) [inline web-client/integration-tests/adcClerkViewsSectionMessagesJourney.test.js]: Is this necessary if the next step is to log in as a different user?

File: web-client/integration-tests/journey/createNewMessageOnCase.js [hunk 2]
```diff
@@ -47,12 +56,21 @@ export const createNewMessageOnCase = cerebralTest => {
...
     );
 
-    cerebralTest.testMessageSubject = `what kind of bear is best? ${Date.now()}`;
-
-    await cerebralTest.runSequence('updateModalFormValueSequence', {
-      key: 'subject',
-      value: cerebralTest.testMessageSubject,
-    });
+    if (subject) {
+      cerebralTest.testMessageSubject = subject;
```
Comment (rachelschneiderman) [inline web-client/integration-tests/journey/createNewMessageOnCase.js]: Could we dry up this section by checking for the existence of the subject override? 

`cerebralTest.testMessageSubject = subject || `what kind of bear is best? ${Date.now()}``

File: web-client/integration-tests/journey/getUserMessageCount.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const getUserMessageCount = async (cerebralTest, box, queue) => {
```
Comment (rachelschneiderman) [inline web-client/integration-tests/journey/getUserMessageCount.js]: I really like pulling this into a named file! Should this be moved into a "helpers" folder so we can add more functions like this in the same place in the future?

File: web-client/src/presenter/actions/setDefaultTableSortAction.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * sets the default sort based on the section picked.
+ *
+ * @param {object} providers the providers object
+ * @param {Function} providers.get the cerebral get function
+ * @param {object} providers.props the props passed to the sequence
+ * @param {object} providers.store the cerebral store object
+ */
+export const setDefaultTableSortAction = ({ get, props, store }) => {
+  // Story 9490 requires the sorting to only apply to the ADC Section boxes.  So these if statements exist to limit the scope for now.  These if statements can be removed in future stories that expand the scope.
```
Comment (rachelschneiderman) [inline web-client/src/presenter/actions/setDefaultTableSortAction.js]: Y'all wrote some amazing tests that cover exactly what this comment is saying! Can this comment be removed in favor of using the tests as docs?

Comment (ujahio) [reply web-client/src/presenter/actions/setDefaultTableSortAction.js]: Removed the comment!

File: web-client/integration-tests/journey/getUserMessageCount.js (hunk 3 above)
Comment (ujahio) [reply web-client/integration-tests/journey/getUserMessageCount.js]: I agree.. its been moved to the helper file

File: web-client/integration-tests/journey/createNewMessageOnCase.js (hunk 2 above)
Comment (ujahio) [reply web-client/integration-tests/journey/createNewMessageOnCase.js]: Good idea.. Done!

File: web-client/integration-tests/adcClerkViewsSectionMessagesJourney.test.js (hunk 1 above)
Comment (ujahio) [reply web-client/integration-tests/adcClerkViewsSectionMessagesJourney.test.js]: Good catch.. not necessary at all..

**Pull request closed without merging**